CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## 0.0.1-alpha.7
+
+### Bug fixes
+
+- Implement Python's floor division semantics for `Literal` `int`s ([#18249](https://github.com/astral-sh/ruff/pull/18249))
+- Don't warn `yield` not in function when `yield` is in function ([#18008](https://github.com/astral-sh/ruff/pull/18008))
+- Fix attribute writes to unions/intersections including modules ([#18313](https://github.com/astral-sh/ruff/pull/18313))
+- Fix binary intersection comparison inference logic ([#18266](https://github.com/astral-sh/ruff/pull/18266))
+- Fix instance vs callable subtyping/assignability ([#18260](https://github.com/astral-sh/ruff/pull/18260))
+- Ignore `ClassVar` declarations when resolving instance members ([#18241](https://github.com/astral-sh/ruff/pull/18241))
+- Infer types for `ty_extensions.Intersection[A, B]` tuple expressions ([#18321](https://github.com/astral-sh/ruff/pull/18321))
+- Respect `MRO_NO_OBJECT_FALLBACK` policy when looking up symbols on `type` instances ([#18312](https://github.com/astral-sh/ruff/pull/18312))
```

Comment:
sort of an internal change?

```suggestion
```

Reply:
I mean, it's not the most impactful change in the world, but it's not internal. It even has some ecosystem changes.